backend/code_review_backend/issues/api.py

```diff
@@ -93,7 +95,9 @@ class DiffViewSet(viewsets.ReadOnlyModelViewSet):
 
     def get_queryset(self):
         diffs = (
-            Diff.objects.all()
+            Diff.objects
+            # Because of the perf. hit filter issues that are not older than today - 3 months.
+            .filter(created__gte=date.today() + timedelta(days=90))
```

Comment:
It should be `- timedelta`, otherwise you'll only look for "future" issues :P

Reply:
Yes, I've seen it after I've pushed it 👍